pwndbg/color/lexer.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import re
+
+import six
+from pygments.lexer import RegexLexer
+from pygments.lexer import include
+from pygments.token import Comment
+from pygments.token import Name
+from pygments.token import Number
+from pygments.token import Operator
+from pygments.token import Other
+from pygments.token import Punctuation
+from pygments.token import String
+from pygments.token import Text
+
+__all__ = ['PwntoolsLexer']
+
+# Text        Token.Text            for any type of text data
+# Whitespace  Token.Text.Whitespace for specially highlighted whitespace
+# Error       Token.Error           represents lexer errors
+# Other       Token.Other           special token for data not matched by a parser (e.g. HTML markup in PHP code)
+# Keyword     Token.Keyword         any kind of keywords
+# Name        Token.Name            variable/function names
+# Literal     Token.Literal         Any literals
+# String      Token.Literal.String  string literals
+# Number      Token.Literal.Number  number literals
+# Operator    Token.Operator        operators (+, not...)
+# Punctuation Token.Punctuation     punctuation ([, (...)
+# Comment     Token.Comment         any kind of comments
+# Generic     Token.Generic         generic tokens (have a look at the explanation below)
+class PwntoolsLexer(RegexLexer):
+    """
+    Fork foom pwntools
```

Comment:
Typo: `foom` -> `from`

Reply:
Can we also add a link to the file in pwntools repo?

Reply:
What do you means a **link**? I put the url at the next line. Or you mean something like git submodule? 
But I have some changes and remove some useless part.

Reply:
Fixed the typo